crates/forge/tests/data/forking/src/lib.cairo

```diff
@@ -91,7 +88,7 @@ mod tests {
     }
 
     #[test]
-    #[fork(url: "{{ NODE_RPC_URL }}", block_id: BlockId::Tag(Latest))]
+    #[fork(url: "{{ NODE_RPC_URL }}", block_tag: latest)]
```

Comment:
In runner code we have an error in code that says that only accepted tag is `Latest` and here we use lowercase. What's the actual error that is displayed to the user if tag other than `latest` is use?

Reply:
Can you link it?

Reply:
This actual error so everything is correct:
```
error: Plugin diagnostic: Both options failed
First variant: #[fork] <block_tag> value incorrect, expected: latest
Second variant: #[fork] can be used with unnamed attributes only
Resolve at least one of them
```

Reply:
Here https://github.com/foundry-rs/starknet-foundry/blob/6e036fc90aeffd026e8a8ef009709ec01ebf8521/crates/forge/src/run_tests/resolve_config.rs#L89

Maybe it's get converted at some point, or it's a different thing and I'm just confused?

Reply:
https://github.com/foundry-rs/starknet-foundry/blob/6e036fc90aeffd026e8a8ef009709ec01ebf8521/crates/forge/src/run_tests/resolve_config.rs#L92

Reply:
Well this is different as it read configuration form `Scarb.toml` , but i agree it should be `latest`. What's weird it is fixed in docs. Anyway changed